src/init.tbasicu

```diff
@@ -1,18 +1,61 @@
+#Region "TBGL Global Settings"
+
+tbgl_UseTexturing %TRUE
+tbgl_UseAlphaTest %TRUE
+
+#EndRegion
+
+#Region "title"
+
+%SIDE = 4
+
+%CTR = 50
+%CTG = 0
+%CTB = 0
+
+%CSR = 0
+%CSG = 0
+%CSB = 50
+
+%MAXBULLES = 30
+%GROUND = 5
+%BOOST = 600
+%TIMEBOOST = 400
+%TIMETITLE = 650
+
+$TITLEPNG = "\media\BTB1024.png"
+$TITLESPRITEPNG = "\media\bulle512.png"
+
+
+type tBulles
```

Comment:
I think there is a minor typo `tBulles` could be `tBubble`? Or maybe it is in French? I would prefer to stick with English. What do you think?

Reply:
yes FR, first thing that come in mind to name it 
OK for EN,  I'll refactor the script when finished